mypy_boto3_builder/stubs_static/boto3/session.pyi

```diff
@@ -44,7 +44,7 @@ class Session:
         partition_name: str = "aws",
         allow_non_regional: bool = False,
     ) -> List[str]: ...
-    def get_credentials(self) -> Credentials: ...
+    def get_credentials(self) -> Credentials | None: ...
```

Comment:
Use `Optional[Credentials]` for `py37` support.

Reply:
Oops, thanks, I meant to see about doing that before submitting this.